text/0192-multi-search-api.md

```diff
@@ -66,7 +66,34 @@ Each element of this array is a JSON object with the required field `indexUid`,
 
 Results of the search queries as an array of search results.
 
-Each element of this array is a JSON object representing the results of the search query with the same index in the `queries` array from the request, and an additional `indexUid` field that is equal to the `indexUid` field in that search query.
+Each element in this array contains the results of the search queries in the same order they have been requested. Additionally to the [usual fields returned by a search result](./0118-search-api.md#31-formatting-search-results), an `indexUid` field is present with the index UID on which the search has been performed.
+
+example:
+
+Search queries:
+```json
+{
+   "queries": [ { "indexUid": "movie", "q": "wonder" }, { "indexUid": "books", "q": "king" } ]
+}
+'''
```

Comment:
These are the wrong quotes. I don't manage to make a suggestion with the correct ones

Reply:
````suggestion
```
````

wow didn't notice they were escaped ticks rather than backticks in your initial comment 😅 

this should fix it